tests/test_adapter.py

```diff
@@ -25,7 +25,7 @@ def configured_adapter():
 
 
 def test_asdict_all_fields():
-    product = Product(**_PRODUCT_ALL_KWARGS)
+    product = Product(**_PRODUCT_ALL_KWARGS)  # type: ignore[arg-type]
```

Comment:
Not quite satisfied in adding `# type: ignore[arg-type]` to suppress the mypy errors here. Nonetheless, it's needed because mypy is strict in the order of the constructor arguments and `_PRODUCT_ALL_KWARGS` doesn't follow the order.

Reply:
> Not quite satisfied in adding # type: ignore[arg-type] to suppress the mypy errors here. Nonetheless, it's needed because mypy is strict in the order of the constructor arguments and _PRODUCT_ALL_KWARGS doesn't follow the order.

What do you mean by "order" @BurnzZ?

Reply:
My bad @kmike I got confused with **mypy**'s errors like:
```
error: Argument 1 to "Foo" has incompatible type "**Dict[str, object]"; expected "int"
error: Argument 1 to "Foo" has incompatible type "**Dict[str, object]"; expected "str"
```

I fixed this by correctly annotating the definitions like `_PRODUCT_ALL_KWARGS` instead. https://github.com/zytedata/zyte-common-items/pull/10/commits/36bd3ac8548144f81d3b9b21cf92b1c9922784d9